Makefile

```diff
@@ -40,7 +40,7 @@ apply: verify build ## Deploy the kwok controller from the current state of your
 		--set-string controller.env[0].value=true
 
 delete: ## Delete the controller from your ~/.kube/config cluster
-	helm uninstall karpenter --namespace ${KARPENTER_NAMESPACE}
+	helm uninstall karpenter --namespace kube-system
```

Comment:
Seems like we should just update both the `make apply` and the `make deploy` targets to use `$KARPENTER_NAMESPACE` and we should change `$SYSTEM_NAMESPACE` to `$KARPENTER_NAMESPACE`

Reply:
Yes, we can do that too instead of hardcoding.